src/extensions/prisma-query/find-queries.ts

```diff
@@ -9,6 +9,16 @@ export type PrismaQuery = {
   args: (string | Record<string, any>)[];
 };
 
+const maybeReplaceModelQueryBackticks = (
+  operation: PrismaQuery["operation"],
+  args: PrismaQuery["args"]
+): PrismaQuery["args"] => {
+  if (!operation.includes("$")) {
```

Comment:
I dont know the context that well. 

Do we need the includes check here? Could we just always run a replace on args?

If yes, then this util could be renamed to `replaceBackticks` 🤔

Reply:
Just to add to the comment, if it is really necessary to check wether the operation has `$`, we could still rename this to simple `replaceBackticks` and use it only if we need to - meaning, we could move `if (!operation.includes("$")) {` outside of the method?

Maybe it could look like this:
```
const replaceBackticks = (
  list: string[]
): PrismaQuery["args"] => {
return list.map(value => value.replace(/`/g, '"'));
}
```

Reply:
Yeah so we need to check the operation because there's a difference between a model query (.findMany, .create, etc.) and a generic query ($transaction, $connect, $queryRaw, $executeRaw). Generic queries - well the ones that take arguments - are allowed backticks, and some actually requrire them (`queryRaw`) so we don't want to change back ticks in those cases

Reply:
Generic queries are also sent to the engine differently than these model queries, so even if they contain model queries inside them (e.g. in a `$transaction`), the back ticks should still be fine inside. 

Note that I have yet to test this theory about backticks inside transactions, because I can't get transactions in text-editors to work at all in dev environment or prod.

Reply:
I'll move the check outside of the function though.

Reply:
Btw, what if the backticks are used on purpose with a var inside like this `${value}` ? Is that a possible scenario?

Reply:
@bonzzy good question. 

I just tried it in prod and this is what happens: 
<img width="1280" alt="Screenshot 2022-11-30 at 11 09 06" src="https://user-images.githubusercontent.com/55739812/204768178-bb21e189-124f-49e9-a9ac-7e576e192e58.png">
It looks like variable substitution doesn't even work

Direct variable interpolation also doesn't work, this time throwing an error

<img width="1396" alt="Screenshot 2022-11-30 at 11 12 50" src="https://user-images.githubusercontent.com/55739812/204768988-b67b68e9-fe9f-49f8-85df-e95b48646aa4.png">

Reply:
@bonzzy by the point the code I changed is reached, the variables should technically already be evaluated/defined, at least in theory, right?